scripts/import_osm.py

```diff
@@ -0,0 +1,111 @@
+import argparse
+import sys
+import osmnx as ox
+from ipygis import get_connection_url
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.schema import CreateSchema
+from geoalchemy2 import Geometry, WKTElement
+from typing import Dict
+
+# test simple import now, convert to module later
+sys.path.insert(0, "..")
+from models import OSMPoint
+from osm_tags import tags_to_filter
+
+
+class OsmImporter(object):
+    """Import OSM data to PostGIS using Overpass API."""
+
+    def __init__(self, args: Dict):
+        args = self._parse_args(args)
```

Comment:
Good idea to separate the parsing method. Probably should unify this for all importers in the future, because all the importers do quite the same thing.